server/index.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+
+const app = express();
+const port = 3000;
+const controller = require('./controller.js');
+
+// middleware
+
+app.use(express.json());
+
+// modle
+app.get('/', controller.getPageInfo);
```

Comment:
Nice work implementing the MVC here, good separation of concerns, good dry implementation.